### PR TITLE
Builders>appchain devs/dapp devs reorg

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Reorganize the Builders section so that it contains content specific to building an Appchain, and move all content specific to what developers can build on top of an Appchain to a new section, the Explore section.

This PR also merges the Substrate and EVM Subsquid page into one "Quick Start" page under Explore > Integrations > Indexers > Subsquid

Redirects PR: https://github.com/papermoonio/tanssi-mkdocs/pull/30

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: explore/developer-toolkit/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Developer Toolkit
+description: TODO
```
Comment (augustocollado) [inline explore/developer-toolkit/index.md:3]:
```suggestion
description: Learn to interact with your Tanssi appchain using tools like Chopsticks, Polkadot.js, Sidecar API, and Remix, Hardhat, Foundry (if Ethereum compatible).
```

File: explore/integrations/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Integrations
+description: TODO
```
Comment (augustocollado) [inline explore/integrations/index.md:3]:
```suggestion
description: Learn about the integrations that allow you to index and query Ethereum data on Your Tanssi EVM Appchain and the Oracles to provide asset price feeds.
```

**Review by augustocollado — CHANGES_REQUESTED** [review]
It looks great!
I feel that there must be a better name than "Explore" for the section, but I couldn't come up with a better one :P
(Oh, and there's some conflicts to fix)

**Review by albertov19 — CHANGES_REQUESTED** [review]
I agree with @augustocollado that maybe "Explore" is not the best name for it. But I'm also unsure what is the right name.

"Appchain API" is the right way to express it to me, but maybe a different word would be better.

I want to put this section next to "Builders," tho because it is not like:

"Section on Appchains" | "Section for Infra" | "Philosophical Section" | "Section around Appchains"

**Review by augustocollado — DISMISSED** [review]
Looks great!

**Review by albertov19 — DISMISSED** [review]
LGTM

**Review by themacexpert — APPROVED** [review]
LGTM  👍

**Pull request merged**